core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -170,6 +172,15 @@ sealed trait ZValidation[+W, +E, +A] { self =>
   final def toEither[E1 >: E]: Either[NonEmptyChunk[E1], A] =
     fold(Left(_), Right(_))
 
+  /**
+   * Transforms this `ZValidation` to an `Either`, discarding the order in which the errors occurred and discarding the log.
+   */
+  final def toEitherUnordered[E1 >: E]: Either[NonEmptyMultiSet[E], A] =
```

Comment:
`ToEitherMultiSet`? We technically aren't guaranteeing any ordering for the errors with the `Chunk` variant, we just actually observe them in some order and happen to return them in the same order.

Reply:
Renamed :+1: